about.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Meal Finder App</title>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://cdn.linearicons.com/free/1.0.0/svgembedder.min.js"></script>
+    <script type="text/javascript" src="script.js"></script>
+    <script src="https://cdn.linearicons.com/free/1.0.0/svgembedder.min.js"></script>
+  </head>
+
+  <body>
+    <!-- Top Navigation Menu -->
+    <nav class="topnav">
+      <div class="active">
+        <a href="#home"
+          >Mealoftheday.com
+          <svg class="lnr lnr-dinner">
+            <use xlink:href="#lnr-dinner"></use>
+          </svg>
+        </a>
+      </div>
+      <div id="myLinks">
+        <a href="about.html">About us</a>
+        <a href="#contact">Contact us</a>
+        <a href="#about">Skills</a>
+      </div>
+      <a href="javascript:void(0);" class="icon" onclick="myFunction()">
```

Comment:
`myFunction` is not a proper name for function. Could you please come up with something more descriptive?)

Reply:
fair enough. I've changed it to "burgermenu".